src/main/java/net/openhft/chronicle/threads/BlockingEventLoop.java

```diff
@@ -43,34 +44,42 @@ public class BlockingEventLoop extends AbstractLifecycleEventLoop implements Eve
     @NotNull
     private transient final ExecutorService service;
     private final List<EventHandler> handlers = new CopyOnWriteArrayList<>();
+    private final List<Runner> runners = new CopyOnWriteArrayList<>();
     private final NamedThreadFactory threadFactory;
-    private final Pauser pauser;
+    private final Supplier<Pauser> pauserSupplier;
```

Comment:
I started doing this work ages ago, but stopped short of implementing it for BlockingEventLoop apparently